tasmota/tasmota_xsns_sensor/xsns_79_as608.ino

```diff
@@ -172,6 +172,8 @@ void As608Loop(void) {
     p = As608Finger->fingerSearch();      // Match found
     if (p != FINGERPRINT_OK) {
 //      AddLog(LOG_LEVEL_DEBUG, PSTR("AS6: No matching finger"));
+      Response_P(PSTR("{\"" D_JSON_FPRINT "\":\"NOMATCH\"}"), As608.finger_id, As608.confidence);
```

Comment:
Too much copy and paste!

Replace by this:
```
Response_P(PSTR("{\"" D_JSON_FPRINT "\":\"NOMATCH\"}"));
```

Reply:
🤦  sorry for the noobiness